tests/cong.test.cc

```diff
@@ -391,7 +391,8 @@ TEST_CASE("Congruence 11: congruence on big finite semigroup",
   std::vector<relation_t> extra(
       {relation_t({0, 3, 2, 1, 3, 2, 2}, {3, 2, 2, 1, 3, 3})});
   Congruence cong("twosided", &S, extra);
-  cong.set_report(CONG_REPORT);
+  cong.set_report(true);
+  cong.force_tc_prefill();
```

Comment:
I'm guessing these 3 lines were staged accidentally?  Forced tests go in other files, and I'm guessing we still want to use `CONG_REPORT`.